src/kata3.numberToReversedDigits.js

```diff
@@ -1,3 +1,4 @@
-const numberToReversedDigits = number => {};
 
+const numberToReversedDigits = number => 
+number.toString().split('').reverse().map(Number);
```

Comment:
Great solution. Loving the implicit return. Also the use of map by passing the number constructor function is quite advanced in my opinion. Do you understand how it works? Brilliant if you do :D